packages/billing/tsup.config.mjs

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+    entry: ['src/**/*.ts', '!./src/**/*.d.ts', '!./src/**/*.spec.ts'],
```

Comment:
I guess we should take it dynamically from `package.json` exports field

Reply:
Why? For now, this config is enough.

Reply:
to be in sync with exports.

Exports explicitly defining what your package is exporting, and here you have a separate config which can differs from exports

Reply:
Done